src/components/AssetManagementPanel/AssetManagementForm/FormVariants/EditableAssetTitle/EditableAssetTitle.tsx

```diff
@@ -10,9 +10,17 @@ interface EditableAssetTitleProps {
   isEditable: boolean;
   newValue?: string;
   onSetNewValue?: (newBeneficiary: string) => void;
+  errorState?: string;
 }
 
-export const EditableAssetTitle = ({ role, value, newValue, isEditable, onSetNewValue }: EditableAssetTitleProps) => {
+export const EditableAssetTitle = ({
+  role,
+  value,
+  newValue,
+  isEditable,
+  onSetNewValue,
+  errorState,
```

Comment:
```suggestion
  changeHolderState,
```

Reply:
i understand but i think keeping it dynamic would be better cos of the beneficiary part, they are using the same component.

Reply:
Oh I misread. Okay so in this case, this component need not need to know of the context of it's parent (ie the different enum for state). Instead it can just be a single prop `error?: string` which will show that errror message if it exist and if it doesnt it is not in the error state. 

If not we are makign this EditableAssetTitle doing the work of it's parent.

Reply:
hmmm.. ok will update accordingly